Improve the README

ixxie opened the pull request:
A new README for Threlte 6.

Animated banner scoped our because its hard to get it right.

grischaerbe commented on README.md:
```diff
@@ -1,38 +1,75 @@
+Threlte is a [Svelte](https://svelte.dev/) library that simplifies creating 3D apps for the web. It provides a **declarative**, **type-safe**, **reactive** and **interactive** API out-of-the-box.
 
-## Official Documentation
+Threlte's core **3D rendering** is powered by [Three.js](https://threejs.org/), but its also provides a **physics engine** through [Rapier](https://rapier.rs/) and an **animation engine** via [Theatre.js](https://www.theatrejs.com/). 
```
Suggestion:

"Threlte's **3D rendering** is powered by [Three.js](https://threejs.org/), it provides a **physics engine** through [Rapier](https://rapier.rs/) and an **animation engine** via [Theatre.js](https://www.theatrejs.com/)."

grischaerbe commented on README.md:
```diff
@@ -1,38 +1,75 @@
+
+ - [Plugins](https://threlte.xyz/docs/learn/advanced/plugins) making it easy to extend Threlte with custom code and logic.
+
+ - [Events](https://threlte.xyz/docs/learn/basics/events) lets you access state transformations in your scenes.
```
This sounds a bit odd, maybe "Events enabling robust and Svelte-native access to state transformations occurring within your scenes." or along these lines?

grischaerbe commented on README.md:
```diff
@@ -1,38 +1,75 @@
+
+ - [Events](https://threlte.xyz/docs/learn/basics/events) lets you access state transformations in your scenes.
+
+ - [Interactivity](https://threlte.xyz/docs/reference/extras/interactivity) makes it easy to drive state with user actions on 3D objects.
```
"… makes it easy to react to user input …"

grischaerbe reviewed (changes requested):
Thanks again, looks and reads so much better than before ❤️ 

Found some smaller things here and there.
Also: Can you replace all links to "https://threlte.xyz" with "https://next.threlte.xyz" so it points to the next page? When merging to main (aka releasing T6) we can get rid of all "next" occurrences in one go 👍

ixxie commented:
@grischaerbe I addressed all the issues, but sometimes in somewhat different ways you suggested. I also streamline some other text a bit to shorten it. Can you give it a read through again to see if you are good with all of it?

grischaerbe commented:
LGTM 🚀 Thank you so much!
